src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IInterpolatedStringOperation.cs

```diff
@@ -4749,7 +4749,7 @@ public void InterpolationEscapeConstantValue_WithoutDefaultHandler()
           null
       IInterpolatedStringTextOperation (OperationKind.InterpolatedStringText, Type: null) (Syntax: ' }}')
         Text:
-          ILiteralOperation (OperationKind.Literal, Type: System.String, Constant: "" }}"", IsImplicit) (Syntax: ' }}')
+          ILiteralOperation (OperationKind.Literal, Type: System.String, Constant: "" }"", IsImplicit) (Syntax: ' }}')
```

Comment:
this test appears to be demonstrating that the bound nodes were in error previously.  For `$""{{ {i} }}""` the literals should only have one brace, not two like the test seemed to indicate.